### PR TITLE
added functionality to share files with definite package, installed on device

### PR DESCRIPTION
**Pull request by red-avtovo**
added functionality to share files with definite package, installed on device
#### APK
[Download build #8770](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8770/artifact/build/artifact/wire-dev-PR804-8770.apk)
[Download build #8772](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8772/artifact/build/artifact/wire-dev-PR804-8772.apk)
[Download build #8774](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8774/artifact/build/artifact/wire-dev-PR804-8774.apk)

### CODE REVIEW

**Comment by red-avtovo** [comment]
Build again

**Comment by hakonbo** [comment]
build again

**Comment by red-avtovo** [comment]
build again

File: testing_gallery/src/main/java/com/wire/testinggallery/SharingCommandReceiver.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    }
+
+    private Intent createFilesShareIntent(String requestedPackage, List<File> files) {
+        Intent shareIntent = createShareIntent(requestedPackage);
+        shareIntent.setAction(Intent.ACTION_SEND_MULTIPLE);
+        shareIntent.putParcelableArrayListExtra(Intent.EXTRA_STREAM, getUrisFromFiles(files));
+        return shareIntent;
+    }
+
+    private ArrayList<Uri> getUrisFromFiles(List<File> files) {
```
Comment (hakonbo) [inline testing_gallery/src/main/java/com/wire/testinggallery/SharingCommandReceiver.java]: Add `//NOPMD` at the end here to disable check for this line.

Comment (red-avtovo) [inline testing_gallery/src/main/java/com/wire/testinggallery/SharingCommandReceiver.java]: Thanks!

**Pull request merged**